app/assets/javascripts/show-hide-content.js

```diff
@@ -13,7 +13,7 @@
     // Radio and Checkbox selectors
     var selectors = {
       namespace: 'ShowHideContent',
-      radio: '.block-label[data-target] input[type="radio"]',
+      radio: '[data-target] > input[type="radio"]',
```

Comment:
For whatever reason other front end apps seem to have this ^^ in the `show-hide-content.js` is `block-label` a legacy class?

Reply:
I do not know.
You could ask in the `#community-frontend` Slack channel